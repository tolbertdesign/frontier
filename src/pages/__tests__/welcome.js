import { shallowMount } from '@vue/test-utils'
import WelcomePage from '@/pages/welcome'

describe('welcome page', () => {
  test('mounts properly', () => {
    const wrapper = shallowMount(WelcomePage)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders properly', () => {
    const wrapper = shallowMount(WelcomePage)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
