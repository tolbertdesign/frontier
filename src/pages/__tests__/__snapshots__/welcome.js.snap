// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`welcome page renders properly 1`] = `
<div id="welcome-page" class="container mx-auto max-w-sm m-6">
  <userloginform-stub msg="ðŸ‘‹ Welcome to Frontier, a themeable design system"></userloginform-stub>
</div>
`;
